Ports: An automation step for generating table with available ports (#17233)
SerenityOS/serenity: Ports/README.md

  that acts as the "clean", patched version of the port that is ready to be built
- The "remote" repository can be pushed to, pulled from and generally anything that
  you'd want to do with a remote repo.

> - After leaving the dev shell, all patches are updated, the user will be prompted
>   whether they wish to generate a new patch readme file and new list with available
>   ports be generated.

Reviewer: ```suggestion
- After leaving the dev shell, all patches are updated and the user will be prompted
  whether they wish to generate a new patch readme file and update the list of
  available ports.
```

Author: changed!

#### `prefix`

> The location of the portsʼ directory, only used for the `package.db` file for

Reviewer 2: no use of fancy quotes, `'` is fine

Author: iʼd say just «quotes» 😅. (unicode — your days are still to come! 😆)

Defaults to `false`.

#### `description`

The description of the port. Currently required in the list of [available ports](#available-ports) only.

> 

Reviewer: I think these `Ports/README.md` changes should not have their own commit, but should be incorporated into the commits with the relevant changes instead.

Author: should be better now!

You can either:

> - Add new ports – just get the software to build and add the necessary patches

Reviewer 2: useless noise, there's hundreds of these across the project - fixing one is not helpful.

Author: well… 🤷‍♂️
